apps/web/src/styles/theme.ts

```diff
@@ -200,6 +262,7 @@ const theme = extendTheme({
         },
       },
       ":root": {
+        "--chakra-colors-black": mode(dark.grey.white, light.grey.white)(props),
```

Comment:
?? black is white?

Reply:
black is white in the light mode and vice versa 🤪